### PR TITLE
pick up the latest changes for MobileSDK-Windows repo

### PR DESCRIPTION
**Pull request by helenren**

### CODE REVIEW

File: src/windows/com.salesforce.plugin.oauthProxy.js [hunk 1]
```diff
@@ -24,7 +24,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-var SALESFORCE_MOBILE_SDK_VERSION = "4.0.0";
+var SALESFORCE_MOBILE_SDK_VERSION = "4.2.0";
```
Comment (bhariharan) [inline src/windows/com.salesforce.plugin.oauthProxy.js]: This should be `5.0.0` everywhere.

File: src/windows/com.salesforce.plugin.smartstoreProxy.js [hunk 2]
```diff
@@ -24,7 +24,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-    var SALESFORCE_MOBILE_SDK_VERSION = "4.0.0";
+    var SALESFORCE_MOBILE_SDK_VERSION = "4.2.0";
```
Comment (bhariharan) [inline src/windows/com.salesforce.plugin.smartstoreProxy.js]: `5.0.0`.

**Comment by bhariharan** [comment]
@helenren I was under the impression `Cordova` was broken on `Windows`? Do we need to update this, if that's the case?

File: src/windows/com.salesforce.plugin.smartstoreProxy.js (hunk 2 above)
Comment (helenren) [inline src/windows/com.salesforce.plugin.smartstoreProxy.js]: haven't bump the version to 5.0 on mobilesdk-windows yet.

**Comment by helenren** [comment]
At before it was. Urvin got the latest feedback from Microsft that the issue has been fixed and it can work on their side. That's why Urvin want me to have a try to verify/know if it works or not.

File: src/windows/com.salesforce.plugin.smartstoreProxy.js (hunk 2 above)
Comment (bhariharan) [inline src/windows/com.salesforce.plugin.smartstoreProxy.js]: Doesn't matter. `unstable` has the upcoming version number, not the current prod one.

**Comment by bhariharan** [comment]
@helenren Could you verify first on your fork, before updating the main repo?

**Comment by helenren** [comment]
@bhariharan sure, i can. While, wondering that why we should keep the codes here far behind for Windows.

**Pull request closed without merging**